src/Classes/Fighter/Fighter.ts

```diff
@@ -112,11 +112,14 @@ export class Fighter extends PlayerClass {
 
   level5(pc: PlayerCharacter, params: LevelingParams): void {
     this.pushFighterFeatures(pc, 5);
-    pc.pcHelper.addResourceTraits({
-      title: "Extra Attack",
-      description: "Number of Extra Attacks you can make.",
-      resourceMax: { value: 1 },
-    });
+    //check for extra attack already
```

Comment:
We can blindly add this because Fighter is the only class to have an extra attack as a resource trait